dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/internal/domain/TracingFileStrategyTest.kt

```diff
@@ -0,0 +1,136 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-2019 Datadog, Inc.
+ */
+
+package com.datadog.android.tracing.internal.domain
+
+import com.datadog.android.core.internal.data.Reader
+import com.datadog.android.core.internal.data.Writer
+import com.datadog.android.core.internal.data.file.DeferredWriter
+import com.datadog.android.core.internal.domain.PersistenceStrategy
+import com.datadog.android.core.internal.threading.LazyHandlerThread
+import com.datadog.android.log.internal.domain.FilePersistenceStrategyTest
+import com.datadog.android.utils.extension.assertMatches
+import com.datadog.android.utils.forge.Configurator
+import com.datadog.android.utils.forge.SpanForgeryFactory
+import com.datadog.tools.unit.invokeMethod
+import com.google.gson.JsonObject
+import com.google.gson.JsonPrimitive
+import datadog.opentracing.DDSpan
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import io.opentracing.Tracer
+import java.io.File
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+
+@ForgeConfiguration(Configurator::class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+internal class TracingFileStrategyTest :
+    FilePersistenceStrategyTest<DDSpan>(
+        TracingFileStrategy.TRACES_FOLDER,
+        modelClass = DDSpan::class.java
+    ) {
+
+    // region LogStrategyTest
+
+    override fun getStrategy(): PersistenceStrategy<DDSpan> {
+        return TracingFileStrategy(
+            context = mockContext,
+            recentDelayMs = RECENT_DELAY_MS,
+            maxBatchSize = MAX_BATCH_SIZE,
+            maxLogPerBatch = MAX_MESSAGES_PER_BATCH,
+            maxDiskSpace = MAX_DISK_SPACE
+        )
+    }
+
+    override fun setUp(writer: Writer<DDSpan>, reader: Reader) {
+        // add fake data into the old data directory
+        val oldDir = File(tempDir, TracingFileStrategy.DATA_FOLDER_ROOT)
+        oldDir.mkdirs()
+        val file1 = File(oldDir, "file1")
+        val file2 = File(oldDir, "file2")
+        file1.createNewFile()
+        file2.createNewFile()
+        assertThat(oldDir).exists()
+        (testedWriter as DeferredWriter<DDSpan>).deferredHandler = mockDeferredHandler
+        (testedWriter as DeferredWriter<DDSpan>).invokeMethod(
+            "consumeQueue",
+            methodEnclosingClass = LazyHandlerThread::class.java
+        ) // consume all the queued messages
+    }
+
+    @Test
+    fun `migrates the data from v0 to v1`() {
+    }
```

Comment:
Missing `// TODO` with `JIRA` reference.

Reply:
that method should be empty...I forgot to ignore it actually and add a reason. We don't have any migration needed for Traces so far.